Mathlib/RingTheory/PowerSeries/Basic.lean

```diff
@@ -166,6 +166,10 @@ theorem ext_iff {φ ψ : R⟦X⟧} : φ = ψ ↔ ∀ n, coeff R n φ = coeff R n
   ⟨fun h n => congr_arg (coeff R n) h, ext⟩
 #align power_series.ext_iff PowerSeries.ext_iff
 
+instance [Subsingleton R] : Subsingleton R⟦X⟧ := by
```

Comment:
Can you add the double implication? (As a lemma, we still want this instance).

Reply:
Sure, I will add this and then merge. Thanks.